templates/frontend/react/frontend/near-config.js

```diff
@@ -1,5 +1,11 @@
 const CONTRACT_NAME = process.env.CONTRACT_NAME;
 
+if (!CONTRACT_NAME) {
+  const errorMessage = 'ERROR: CONTRACT_NAME not defined. Did you run deploy script (`npm run deploy`)? README.md contains more info about it.';
```

Comment:
Here I would simply say "Did you deploy your contract?", to avoid mixing instructions for the website with instructions for the backend.